rep-2005.rst

```diff
@@ -500,6 +500,31 @@ All items on the list for the next distro should already be released into the ro
   * `urdf/urdf <https://index.ros.org/p/urdf/>`_
   * `urdfdom/urdfdom <https://index.ros.org/p/urdfdom/>`_
 
+* Web
+
+  * `foxglove/ros1 <https://github.com/foxglove/ros1>`_
+  * `foxglove/ros2 <https://github.com/foxglove/ros2>`_
+  * `foxglove/rosbag <https://github.com/foxglove/rosbag>`_
+  * `foxglove/rosbag2 <https://github.com/foxglove/rosbag2>`_
+  * `foxglove/rosbag2-node <https://github.com/foxglove/rosbag2-node>`_
+  * `foxglove/rosbag2-web <https://github.com/foxglove/rosbag2-web>`_
+  * `foxglove/rosmsg <https://github.com/foxglove/rosmsg>`_
+  * `foxglove/rosmsg-serialization <https://github.com/foxglove/rosmsg-serialization>`_
+  * `foxglove/rosmsg2-serialization <https://github.com/foxglove/rosmsg2-serialization>`_
+  * `foxglove/rostime <https://github.com/foxglove/rostime>`_
+  * `foxglove/rtps <https://github.com/foxglove/rtps>`_
```

Comment:
There are a few other examples of this in REP 2005, but overall its pretty rare for a company Github org to host important packages that are part of REP 2005 / that are core ROS packages. Seems to me this would be better to live in a more generic org like ``ros-web`` (not sure if that exists)

Reply:
Yes, I have to agree with this sentiment.  The packages in the `RobotWebTools` and the `rosbridge_suite` organizations are fine, but the ones in the `foxglove` organization don't belong in here (in my opinion).  Also I have concerns about repositories called just `ros2` and `rosbag2` as that will cause instant confusion with the core packages of the same name.

As-is, this list is a -1 from me.  To move forward here, I would suggest cutting this list down to just the ones in the `RobotWebTools` and the `rosbridge_suite` organizations.  I would approve that.

Reply:
Agreed, the only examples I see are the DDS implementations, which I believe deserves some exception due to the nature of how ROS 2 is organized, and webots, which I'm not sure what the justification for that was

Reply:
I agree with unbundling - RobotWebTools repos sound uncontroversial on their own, the other ones can be discussed independently. Do we need to list the individual packages within `rosbridge_suite` or should we just have one list item for the repository `RobotWebTools/rosbridge_suite`?

Reply:
Thanks for the feedback! I will remove the foxglove org packages for now and we can discuss them separately.

I should make it clear that all of the packages under foxglove org are NPM (javascript/typescript) modules, which is why they do not appear in index.ros.org, and why they have names such as `rosbag` (because that is the relevant package name in JS ecosystem). Is there any precedent for including non-apt packages in REP 2005? It seems that folks agree they are useful, but maybe not clear how to include them (either way we can discuss on a separate PR).

> Do we need to list the individual packages within rosbridge_suite or should we just have one list item for the repository RobotWebTools/rosbridge_suite? - @emersonknapp 

I followed what appears to be the existing pattern of having one entry per package (rather than per-repo), but I'm happy to restructure however it is preferred.

Reply:
Update: I have removed the foxglove packages. Some of the RobotWebTools packages are still linking to Github because they are also NPM modules. If it is possible to list NPM modules on index.ros.org, please let me know and I'm happy to update the links.

Reply:
They should still be renamed, life is long and for all we know we'll have a rosbag client for JS and that name should be reserved for it (and its generally confusing if there's something called "rosbag" that isn't actually _rosbag_). What are those project's purposes? Maybe `rosbag_js` if its actually a full rosbag implementation, otherwise, some other appropriately descriptive suffix?

Reply:
> They should still be renamed

Actually my mistake, they already have unique names. For example, https://github.com/foxglove/rosbag2 is published on NPM as [@foxglove/rosbag2](https://www.npmjs.com/package/@foxglove/rosbag2). So we are not squatting on any potential global namespace here. They live under our Github org for now only because they are newish projects and the ros-web (web tools) working group was only created very recently, so we are happy to "incubate" them.

> What are those project's purposes? Maybe `rosbag_js` if its actually a full rosbag implementation

They are complete implementations of rosbag (ros 1 + 2), we use them to read bags in Foxglove Studio. I think today we only support read, but we will add write support in the future to.

> for all we know we'll have a rosbag client for JS and that name should be reserved for it

We would love to contribute these as "official" javascript implementations, in fact that's more or less the direction the ros-web working group is going.

Reply:
Got it, `rosbag_js` seems like a good name for that then, though if just a reader for some period of time `rosbag_reader_js` would be also descriptive (should there be another package `rosbag_writer_js` in the future to add to it). Having some name differentiator (other than namespacing) would be most ideal to reduce confusion, even if they're not going to sit parallel to each other in a package manager for a literal name collision